src/Hl7.Fhir.Core/Rest/EntryToHttpExtensions.cs

```diff
@@ -118,6 +127,17 @@ private static void setBodyAndContentType(HttpWebRequest request, Resource data,
                 // request.WriteBody(CompressRequestBody, bin.Content);
                 request.ContentType = bin.ContentType;
             }
+            else if (isSearch)
+            {
+                string bodyParameters = null;
+                foreach(Parameters.ParameterComponent parameter in ((Parameters)data).Parameter)
+                {
+                    if (!string.IsNullOrEmpty(bodyParameters)) bodyParameters += "&";
+                    bodyParameters += $"{parameter.Name}={parameter.Value}";
+                }
+                body = Encoding.UTF8.GetBytes(bodyParameters);
+                request.ContentType = "application/x-www-form-urlencoded";
```

Comment:
Doesn't the contenttype "x-www-form-urlencoded" imply that the parameters need to be url-encoded?

Reply:
True, committed fix

Reply:
Thanks, great job. Would be really awesome if you could also add a unit test for some special characters like <&>"'äëïo€ etcetera. Don't know if we do this for GET already, but I think we should. However that's another issue.